cmd/create.go

```diff
@@ -278,6 +284,26 @@ func newCreateConfig(args []string, newClient ClientFactory) (cfg createConfig,
 	return
 }
 
+// singleCommand that could be used by the current user to minimally recreate the current state.
+func singleCommand(cmd *cobra.Command, args []string, cfg createConfig) string {
+	var b strings.Builder
+	b.WriteString(os.Args[0])           // process executable
```

Comment:
`b.WriteString(cmd.Root().Name())` would be better I think.

Reply:
Done.